cmd/corald/routes/routes.go

```diff
@@ -112,6 +112,12 @@ func routes(a *app.App) {
 				return "/v1/exec/" + c.Params["query_set"]
 			}))
 
+	// Execute xenia queries directly.
+	a.Handle("GET", "/v1/exec/:query_set", handlers.Proxy(xeniadURL, nil))
+
+	// Send a new query to xenia.
+	a.Handle("PUT", "/v1/query", handlers.Proxy(xeniadURL, nil))
```

Comment:
You're right, I was thinking about another endpoint.  This is right.  Let's also add:

```
POST /v1/exec
```